packages/connection-manager/src/js/Connection/XHRConnection.js

```diff
@@ -0,0 +1,187 @@
+import AbstractConnection from "./AbstractConnection";
+import { CLOSE, OPEN, PROGRESS, ABORT, ERROR, SUCCESS } from "./EventConstants";
+import { CANCELED, INIT, DONE, STARTED } from "./StateConstants";
+
+export const ALLOWED_METHODS = ["GET", "POST", "PUT", "DELETE"];
+
+/**
+ * Basic XHR Connection
+ * @todo add/remove headers
+ */
+export default class XHRConnection extends AbstractConnection {
+  /**
+   * Initializes an Instance of XHRConnection
+   * @constructor
+   * @param {string} url – URL to be fetched
+   * @param {object} options
```

Comment:
Please indicate the these arguments are optional.

```
@param {object} [options]
@param {string} [options.method=GET] – used method
@param {*} [options.body] – payload for request
@param {string} [options.headers] – additional headers (like content-type)
```

http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values